src/main/cpp/SwerveDrive.cpp

```diff
@@ -78,12 +81,12 @@ frc::DifferentialDriveWheelSpeeds SwerveDrive::getDifferentialWheelSpeeds() {
 
 /**
  * Called periodically in Robot.cpp. encapsulates driving, path following, and odometry update
- * @param dx the desired robot velocity in the x direction
- * @param dy the desired robot velocity in the y direction
- * @param dtheta the desired robot rotational velocity
+ * @param vx the desired robot velocity in the x direction
+ * @param vy the desired robot velocity in the y direction
+ * @param vtheta the desired robot rotational velocity
  * @param turretAngle the turret's angle on the robot, used to help calculate odometry
 **/
-void SwerveDrive::Periodic(units::meters_per_second_t dx, units::meters_per_second_t dy, units::radians_per_second_t dtheta, double turretAngle) {
+void SwerveDrive::Periodic(units::meters_per_second_t vx, units::meters_per_second_t vy, units::radians_per_second_t vtheta, double turretAngle) {
 
     //update odometry no matter the state
     odometry_->Update(units::degree_t{navx_->GetYaw()}, getRealModuleStates());

```

Comment:
`getRealModuleStates()` (defined [here](https://github.com/Eaglestrike/2023-RobotCode/blob/master/src/main/cpp/SwerveDrive.cpp#L59L66)) just returns the current state of each module (so the measured speed and angle of the wheel) combined into an array of 4 module states. I'm basically just restating what the comment says; does that make sense? Or should I update the comment?

Odometry works by recording the speed and angle of the wheels at each time step and "integrating" them to determine the robot's position. When the robot is given information about its module states, it can figure out its velocity (speed and direction). It can then use the `change in distance = velocity*time` equation, substituting in the current velocity and the time period between each cycle of the code being run. It then adds that change in position to last cycle's position, and in that manner the robot calculates its position on the field.